protocol/messenger_backup_test.go

```diff
@@ -917,3 +917,36 @@ func (s *MessengerBackupSuite) TestBackupChats() {
 	s.Require().True(ok)
 	s.Require().Equal("", chat.Name)
 }
+
+func (s *MessengerBackupSuite) TestBackupleftCommunity() {
```

Comment:
This test doesn't really test the backup as far as I see. Did you forget to add a piece to the test?

Reply:
maybe I need to dig deeper here. But the bug is about recovering from scratch, an account that was backed up through seed phrase. I simulate this by removing the data folder in Status directory.
What I did instead is to make sure that if the persistence layer has the correct data, then it should be enough for verifying that we have the correct information. wdyt?

Reply:
The test is fine, but it would need to be renamed to `TestLeaveCommunity` and be moved to another file, because it currently doesn't do much about the backup

But even better, there is a test called `TestBackupCommunities` that you could add to. You could create another community there, leave it and confirm that Bob2 doesn't have the left community.

It doesn't require to delete the data dir, it just creates a new messenger with the exact same identity (seed)

Reply:
> The test is fine, but it would need to be renamed to `TestLeaveCommunity` and be moved to another file, because it currently doesn't do much about the backup
> 
> But even better, there is a test called `TestBackupCommunities` that you could add to. You could create another community there, leave it and confirm that Bob2 doesn't have the left community.
> 
> It doesn't require to delete the data dir, it just creates a new messenger with the exact same identity (seed)

that's nice idea 😄  sending another revision, thanks